hassfest fails with module not found for black

Running `hassfest` results in the following error:

```
Run docker run --rm -v /home/runner/work/philips-airpurifier-coap/philips-airpurifier-coap://github/workspace hassfest
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/src/homeassistant/script/hassfest/__main__.py", line 7, in <module>
    from . import (
  File "/usr/src/homeassistant/script/hassfest/application_credentials.py", line 4, in <module>
    import black
ModuleNotFoundError: No module named 'black'
Error: Process completed with exit code 1.
```

Is that a problem of `hassfest` or did I do something wrong?

--- a/hassfest/requirements.txt
+++ b/hassfest/requirements.txt
@@ -1,3 +1,4 @@
+black==22.8.0
 pipdeptree==1.0.0
 stdlib-list==0.7.0
 tqdm==4.48.2

